source/modules/MEI.edittrans.xml

```diff
@@ -40,31 +40,16 @@
       <memberOf key="att.evidence"/>
     </classes>
   </classSpec>
-  <classSpec ident="att.metaMark.anl" module="MEI.edittrans" type="atts">
-    <desc xml:lang="en">Analytical domain attributes.</desc>
-  </classSpec>
-  <classSpec ident="att.metaMark.ges" module="MEI.edittrans" type="atts">
```

Comment:
Same comment -- why is att.metaMark.ges removed?  It brings in needed attributes.

Reply:
No, it was moved to MEI.gestural.xml: https://github.com/music-encoding/music-encoding/pull/1025/files#diff-671f8d721efbb54bb038aba70022baff5baf0d14339ad10412e89ac1604b61c1R396

Reply:
the anl part accordingyl to MEI.analytical.xml

Reply:
@pe-ro can we consider this settled then?

Reply:
Yes, as long as `<metaMark>` has the necessary .ges attributes.